code/src/components/TaskList.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import styled from 'styled-components';
```

Comment:
If you change to 'styled-components/macro' your styled components get their names written in the inspector instead of a random combination of letters and numbers :)